docs/core/porting/upgrade-assistant-uwp-framework.md

```diff
@@ -41,10 +41,16 @@ This tool currently supports C#, and in most cases the app will require more eff
 
 Additionally, you may choose to wait for the next version of .NET Upgrade Assistant tool before you start migrating your app, because of the current limitations of the tool:
 
-- ApplicationView APIs not supported.
-- AppWindow related APIs not supported (though it tries to generate a warning where possible and deliberately breaks your code so it doesn't compile until you manually fix things).
-- Custom Views supported (For example a `CustomDialog` that extends `MessageDialog` and calls an api incorrectly, it will not be warned about or fixed).
-- WinRT Components not supported.
+- `ApplicationView` APIs aren't supported.
+- `AppWindow` related APIs aren't supported.
+
+  The upgrade tries to generate a warning where possible, and deliberately breaks your code so it doesn't compile until you adjust your code.
+
+- Custom views aren't supported.
```

Comment:
Should this be are supported or aren't supported?

Reply:
No, the email request was to correct this, that they're not supported 😁